### ISSUE TITLE
False negative when using keyword argument for `NamedTuple`

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5699]
The `NamedTuple` function does not support keyword parameters, so the following causes a runtime error. Pyright doesn't catch this currently.

```python
from typing import NamedTuple

NT1 = NamedTuple(typename="NT1", fields=[("a", str), ("b", str)])
```

This is related to mypy issue https://github.com/python/mypy/issues/8651.

Related to this, the `namedtuple` function does support keyword parameters, but pyright doesn't currently validate those.

**Comment by erictraut** [comment]
This is included in pyright 1.1.324, which I just published. It will also be included in a future release of pylance.
